app/src/main/java/ar/com/wolox/android/example/ui/example/login/LoginPresenter.java

```diff
@@ -1,49 +1,80 @@
 package ar.com.wolox.android.example.ui.example.login;
 
+import android.support.annotation.Nullable;
 import android.text.TextUtils;
 import android.util.Patterns;
+import java.util.List;
 import javax.inject.Inject;
+import ar.com.wolox.android.example.model.User;
+import ar.com.wolox.android.example.network.UserService;
 import ar.com.wolox.wolmo.core.presenter.BasePresenter;
+import ar.com.wolox.wolmo.networking.retrofit.RetrofitServices;
+import ar.com.wolox.wolmo.networking.retrofit.callback.NetworkCallback;
+import okhttp3.ResponseBody;
 
 /**
  *
  */
-class LoginPresenter extends BasePresenter<ILoginView> {
+public class LoginPresenter extends BasePresenter<ILoginView> {
+
+    @Inject RetrofitServices mRetrofitService;
 
     @Inject LoginPresenter() {}
 
     void onLoginValidation(String email, String password) {
```

Comment:
We can add Nullabilities here too. We know that email and password will be not null so @Nonnull string email and @Nonnull String password can be included as parameters

Reply:
✔ **Done**